url.bs

```diff
@@ -62,12 +62,18 @@ might increase in scope somewhat.
 
 <p>This specification depends on the Infra Standard. [[!INFRA]]
 
-<p>Some terms used in this specification are defined in the
-DOM, Encoding, IDNA, and Web IDL Standards.
-[[!DOM]]
-[[!ENCODING]]
-[[!IDNA]]
-[[!WEBIDL]]
+<p>Some terms used in this specification are defined in the following standards and specifications:
+
+<ul class=brief>
+ <li>DOM Standard [[!DOM]]
+ <li>Encoding Standard [[!ENCODING]]
+ <li>File API [[!FILEAPI]]
+ <li>HTML Standard [[!HTML]]
+ <li>Media Source Extensions [[!MEDIA-SOURCE]]
+ <li>Media Capture and Streams [[!MEDIACAPTURE-STREAMS]]
+ <li>Unicode IDNA Compatibility Processing [[!IDNA]]
+ <li>Web IDL [[!WEBIDL]]
+</ul>
```

Comment:
I wouldn't mind if we are using the approach taken by the [HTML Standard](https://html.spec.whatwg.org/#dependencies) where the [terms defined by reference](https://url.spec.whatwg.org/#index-defined-elsewhere) is part of the prose, though I can imagine this being additional work to keep up to date.

But yes, I do think it is a good idea to inform the reader of where definitions of terms can be found in a chronological order -- i.e. before the terms are actually used.